app/controllers/user/reservations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class User::ReservationsController < ApplicationController
+  def index
+    if current_user
+      if current_user.host?
+        @listings = User.find_by(slug: params[:slug]).listings
+      elsif current_user.default?
```

Comment:
This will come in handy for when we show a guest users reservations and past trips